### PR TITLE
[chrome-api-jstocxxtest] Test Transmit

### PR DESCRIPTION
**Pull request by ViktoriiaKovalova**

### CODE REVIEW

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js [hunk 1]
```diff
@@ -846,14 +846,63 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
...
+        /*preferredProtocols=*/ {'t1': true},
+        /*expectedProtocol=*/ chrome.smartCardProviderPrivate.Protocol.T1);
+
+    // Reply should contain the application identifier, followed by the status
+    // bytes that denote success: SW1=0x90, SW2=0x00. This corresponds to the
+    // fake reply built by `testing_smart_card_simulation.cc`.
+    expectReportDataResult(
+        /*requestId=*/ 111,
+        new Uint8Array([
+          0x61, 0x5C, 0x4F, 0x0B, 0xA0, 0x00, 0x00, 0x03, 0x08, 0x00, 0x00,
```
Comment (emaxx-google) [inline smart_card_connector_app/src/chrome-api-jstocxxtest.js:883]: Let's create some constant for this. E.g., for the "application identifier", which would be everything except the last bytes "0x90, 0x00".

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js [hunk 2]
```diff
@@ -846,14 +846,63 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
...
+          0x61, 0x6E, 0x64, 0x5F, 0x56, 0x65, 0x72, 0x69, 0x66, 0x69, 0x63,
+          0x61, 0x74, 0x69, 0x6F, 0x6E, 0x5F, 0x43, 0x61, 0x72, 0x64, 0x5F,
+          0x50, 0x1A, 0x68, 0x74, 0x74, 0x70, 0x3A, 0x2F, 0x2F, 0x63, 0x73,
+          0x72, 0x63, 0x2E, 0x6E, 0x69, 0x73, 0x74, 0x2E, 0x67, 0x6F, 0x76,
+          0x2F, 0x6E, 0x70, 0x69, 0x76, 0x70, 0x90, 0x00
+        ]).buffer,
+        'SUCCESS');
+
+    // Send the SELECT command (the format is per NIST 800-73-4).
+    const selectCommand = new Uint8Array([
```
Comment (emaxx-google) [inline smart_card_connector_app/src/chrome-api-jstocxxtest.js:896]: nit: This is a "real" constant, so `SELECT_COMMAND` would be appropriate spelling for it.

**Review by emaxx-google — APPROVED** [review]

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js (hunk 1 above)
Comment (ViktoriiaKovalova) [reply smart_card_connector_app/src/chrome-api-jstocxxtest.js]: done

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js (hunk 2 above)
Comment (ViktoriiaKovalova) [reply smart_card_connector_app/src/chrome-api-jstocxxtest.js]: done

**Pull request merged**